### PR TITLE
POD-2199: Allow multiple tables Terra -> TDR Ingest

### PR DESCRIPTION
**Pull request by sahakiann**

### CODE REVIEW

File: python/gcp_workspace_table_to_dataset_ingest.py [hunk 1]
```diff
@@ -114,63 +105,69 @@ def get_args() -> argparse.Namespace:
...
-        test_ingest=TEST_INGEST,
-        load_tag=f"{billing_project}_{workspace_name}-{dataset_id}",
-        file_list_bool=False
-    ).run()
+    for terra_table_name in terra_tables:
+        target_table_name = terra_table_name
+
+        # Get sample metrics from Terra
+        sample_metrics = terra_workspace.get_gcp_workspace_metrics(entity_type=terra_table_name)
+        entity_metrics = terra_workspace.get_workspace_entity_info()
```
Comment (snovod) [inline python/gcp_workspace_table_to_dataset_ingest.py:119]: Should this be outside of for loop so you don't hit this multiple times?

**Review by snovod — APPROVED** [review]
One small comment. Also, I think could be helpful if FILTER_EXISTING_IDS and batch size was an input to the python and wdl scripts instead of hard coded.

File: python/gcp_workspace_table_to_dataset_ingest.py (hunk 1 above)
Comment (sahakiann) [reply python/gcp_workspace_table_to_dataset_ingest.py:119]: I think we want this for each entity type (i.e. terra_table_name) right?

Comment (sahakiann) [reply python/gcp_workspace_table_to_dataset_ingest.py]: actually the `get_workspace_entity_info` can be called outside of the for loop, you're right! But I think `get_gcp_workspace_metrics` has to be called within, no?

Comment (snovod) [reply python/gcp_workspace_table_to_dataset_ingest.py]: Yea, getting the individual metrics should happen within the for loop

**Pull request merged**